scripts/setup.sh

```diff
@@ -100,3 +100,47 @@ else
   # Mac OS.
   export CHROME_BIN="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
 fi
+
+# Check the version of Python. If its version does not start with 2.7, then
+# try and see whether the 'python2.7' command exists.
+# Borrowed from bhenning's solution for checking python version
+# http://stackoverflow.com/questions/592620
```

Comment:
I think it's nice to reference this stack overflow article, however I don't think you are checking if the given command exists in this function. There might be some problems with test_python_version if the input command doesn't exist (such as if the user doesn't have Python installed at all).